package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sonar-ui-common",
-  "version": "0.0.48",
+  "version": "0.0.49",
```

Comment:
You should not update the version here yet, I'll show you how to release once the PR is ready.
On the other hand you should update the changelog already.